webapp/src/components/Vault/VaultActionsForm/EarnModal/ActionSteps.tsx

```diff
@@ -67,6 +70,32 @@ const ActionSteps: React.FC<ActionStepsProps> = ({
   } = useV2VaultData(vaultOption);
   const { priceHistory } = useVaultPriceHistory(vaultOption, "earn");
 
+  const asset = useMemo(() => {
+    return vaultOption === "rEARN-stETH" && actionType === "withdraw"
+      ? "stETH"
+      : depositAsset;
+  }, [actionType, depositAsset, vaultOption]);
+
+  const color = getVaultColor(vaultOption);
+  const firstStep = useMemo(() => {
+    if (v2WithdrawOption === "complete") {
+      return STEPS.previewStep;
+    }
+    return STEPS.formStep;
+  }, [v2WithdrawOption]);
+
+  const [minSTETHAmount, setMinSTETHAmount] = useState<BigNumber | undefined>();
+
+  const stETHDepositHelper = useSTETHDepositHelper(vaultVersion);
+
+  const { contract, getMinSTETHAmount } = useLidoCurvePool();
+
+  const [txhash, setTxhash] = useState("");
```

Comment:
Perhaps this should be kept consistent with the state above (`minSTETHAmount`):

`const [txHash, setTxHash] = useState<string | undefined>();`